manifests/gcp_marketplace/schema.yaml

```diff
@@ -92,6 +92,40 @@ x-google-marketplace:
       - apiGroups: ['apiextensions.k8s.io', 'rbac.authorization.k8s.io']
         resources: ['customresourcedefinitions', 'clusterroles', 'clusterrolebindings']
         verbs: ['*']
+    # For CRs in the following deployer-applied Roles:
+    # - kubeflow-pipelines-cache-role
+    # - kubeflow-pipelines-metadata-writer-role
+    # - ml-pipeline
+    # - ml-pipeline-persistenceagent-role
+    # - ml-pipeline-scheduledworkflow-role
+    # - ml-pipeline-viewer-controller-role
+    # - ml-pipeline-ui
+    # - pipeline-runner
+    - type: Role
+      rulesType: CUSTOM
+      rules:
+      - apiGroups: ['argoproj.io']
```

Comment:
I didn't get it, why does deployer need these permissions?

Deployer doesn't deploy these CRs directly.

Reply:
I also didn't get it, Erika suggested the change.